libs/gi/formula/src/data/common/conds.ts

```diff
@@ -0,0 +1,8 @@
+import { allBoolConditionals } from '../util'
+
+export const { enemyFrozen } = allBoolConditionals('static', 'both')
```

Comment:
Why can't these be in resonance.ts?

Reply:
It could, but these conditionals are available outside of resonance too, e.g., if some char's team/self buff depends on whether an enemy is frozen.